server/index.mjs

```diff
@@ -73,6 +67,63 @@ app.get('/api/song/:id/playlists', (req, res) => {
   });
 });
 
+//* ************************* NEW ENDPOINTS **************************** *//
```

Comment:
Very good readability

Reply:
wanted to make it clear what sections I changed for the `CRUD` milestone :)